diff --git a/src/cpp/core/http/Request.cpp b/src/cpp/core/http/Request.cpp
index 76f9a2cde6..9332308669 100644
--- a/src/cpp/core/http/Request.cpp
+++ b/src/cpp/core/http/Request.cpp
@@ -59,8 +59,8 @@ std::string Request::baseUri(BaseUriUse use /*= BaseUriUse::Internal*/) const
    // URI is the same address visible in the browser or the proxy is
    // taking care of the path-rewrite which is something we asked
    // customers to do before v1.4
-   if (rootPath() != kRequestDefaultRootPath)
-      return proxiedUri();
+   return proxiedUri();
+
    if (use == BaseUriUse::Internal)
       return internalUri();
    // When BaseUriUse::External, we omit the internal URI
