commit 89003899fc6382810b6c472cad9238c97f222faf
Author: Ricardo Andrade <ricardo@rstudio.com>
Date:   Thu Aug 20 20:28:24 2020 +0000

    Make auth-none work like any other auth

diff --git a/src/cpp/server/ServerPAMAuth.cpp b/src/cpp/server/ServerPAMAuth.cpp
index 5189c931c1..95f17e8df9 100644
--- a/src/cpp/server/ServerPAMAuth.cpp
+++ b/src/cpp/server/ServerPAMAuth.cpp
@@ -151,10 +151,7 @@ std::string applicationSignInURL(const http::Request& request,
 
 std::string getUserIdentifier(const core::http::Request& request)
 {
-   if (server::options().authNone())
-      return core::system::username();
-   else
-      return core::http::secure_cookie::readSecureCookie(request, kUserIdCookie);
+   return core::http::secure_cookie::readSecureCookie(request, kUserIdCookie);
 }
 
 std::string userIdentifierToLocalUsername(const std::string& userIdentifier)
@@ -262,6 +259,12 @@ void signIn(const http::Request& request,
       return;
    }
 
+   if (server::options().authNone())
+   {
+      auth::handler::setSignInCookies(request, core::system::username(), false, pResponse);
+      pResponse->setMovedTemporarily(request, "./");
+      return;
+   }
 
    std::map<std::string,std::string> variables;
    variables["action"] = applicationURL(request, kDoSignIn);
